### ISSUE TITLE
Hide files on the Remote Tree View

### ISSUE THREAD

**Issue by KronKraken-zz** [icetee/remote-ftp#491]
The Remote Tree view has lots of useless files displayed like `.cache`, `.local`, `.npm`, `.config`, etc. Would be nice to have a file like `.ftphide` so those are not displayed.

**Comment by jimmaaay** [comment]
It'd probably be better to have these as an array in the `.ftpconfig` file. But yes this could be useful for some people.

**Comment by KronKraken-zz** [comment]
Yes could work like that too.

**Comment by KronKraken-zz** [comment]
Any updates?

**Comment by jpxd** [comment]
I don't think this feature will get into the plugin soon as so few people need it and there are so much else issues need to be fixed...

**Comment by hatzopoulos** [comment]
Isn't this covered by the "Show Hidden Files" configuration option? https://github.com/mgrenier/remote-ftp/blob/master/README.md#package-preferences
